composer.json

```diff
@@ -94,6 +94,7 @@
         "drupal/paragraphs": "^1.8",
         "drupal/paragraphs_inline_entity_form": "^1.0@beta",
         "drupal/pathauto": "^1.4",
+        "drupal/permissions_filter": "^1.1",
```

Comment:
I think this should go in as a dev requirement, we're not gonna need that on prod?

Reply:
_makes thinking noises_.... yeah, I guess if perms are controlled in config there's less scope for admins in comparison to a classic D7 site.

Dev requirement it is then 👍

Reply:
All done now with tests passing